### PR TITLE
[[ PostgreSQL SSL ]] Updated dbpostgresql to support secure connections.

### PR DESCRIPTION
**Pull request by livecodemichael**
This pull request relies on https://github.com/livecode/livecode-thirdparty/pull/31

### CODE REVIEW

**Comment by peter-b** [comment]
@livecodemichael: The 64-bit Linux build failed on Travis. Could you please investigate? Thanks. :+1:

**Comment by runrevmark** [comment]
@livecodemichael: I think the third-party submodule pointer hasn't been updated

File: revdb/src/postgresql_connection.cpp [hunk 1]
```diff
@@ -59,11 +61,113 @@ Bool DBConnection_POSTGRESQL::connect(char **args, int numargs)
...
+    const char *t_ssl_opt_key;
+    t_ssl_opt_key = NULL;
+    const char *t_ssl_opt_val;
+    t_ssl_opt_val = NULL;
+    for (int i = 4; i < numargs; i++)
+    {
+        t_delimiter = strchr(args[i], '=');
+        if (t_delimiter != NULL)
+        {
+            *t_delimiter = '\0';
```
Comment (runrevmark) [inline revdb/src/postgresql_connection.cpp]: You shouldn't really do this - the string is a const char \* - it probably shouldn't be mutated ;)

**Comment by runrevmark** [comment]
@livecode-vulcan review ok d05fb6e

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok d05fb6ef91c2458e763ff6747cb8817e87ae2ef1

**Comment by livecode-vulcan** [comment]
😎 test success d05fb6ef91c2458e763ff6747cb8817e87ae2ef1
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/769)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/783)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/770)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/772)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/783)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/793)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/785)

**Pull request merged**